docs/assets/send-message-seq.svg

```diff
@@ -0,0 +1,47 @@
+@startuml
+activate SAS
+SAS -> "API gateway": send message request
+activate "API gateway"
+"API gateway" -> "API Functions": forward message
+activate "API Functions"
+"API Functions" -> "Database": save message
+activate "Database"
+"Database" -> "API Functions": message saved
+deactivate "Database"
+"API Functions" -> "API gateway": return message
```

Comment:
you should use dashed lines (`-->`) for responses, e.g. `"API Functions" --> "API gateway": return message`

Reply:
also, the response from `API Functions` should be after `map message to notification`